frontend/packages/dev-console/src/components/svg/SvgBoxedText.tsx

```diff
@@ -95,6 +105,28 @@ const SvgBoxedText: React.FC<SvgBoxedTextProps> = ({
             : truncateEnd(children, truncate)
           : children}
       </text>
+      {icon && textSize && badgeSize && labelSize && (
+        <>
+          <SvgDropShadowFilter id={iconFilterID} />
+          <rect
+            x={x + labelSize.width / 2 + paddingX - iconSize / 2}
+            y={y}
+            width={iconSize}
+            height={iconSize}
+            fill="#fff"
+            rx={cornerRadius}
+            ry={cornerRadius}
+            filter={createSvgIdUrl(iconFilterID)}
+          />
+          <image
+            x={x + labelSize.width / 2 + paddingX - iconSize / 2 + iconPadding}
+            y={y + iconPadding}
+            width={30}
```

Comment:
Iconsize - iconpadding * 2?

You a seem to be doing a lot computed values but this size remains static even though it is as relative value.